vaadin-side-nav-flow-parent/vaadin-side-nav-testbench/src/main/java/com/vaadin/flow/component/sidenav/testbench/SideNavElement.java

```diff
@@ -56,4 +47,32 @@ public void clickExpandButton() {
         executeScript("arguments[0].click();", element);
     }
 
+    public List<SideNavItemElement> getItems(boolean includeChildren) {
```

Comment:
I think we should keep an overload without the boolean param that returns items without nested children.

Reply:
I could not be sure about the naming. At first, I was thinking of going with something like `getDirectChildItems` and `getItemsRecursively`. Do you have a suggestion?

Reply:
I think the current name is fine, just add an overload and maybe tune the param name to use `nested` and `items`:
```java
List<SideNavItemElement> getItems();
List<SideNavItemElement> getItems(boolean includeNestedItems);
```

Reply:
Done.